markdown/dev/tutorials/pattern-design/adding-measurements/en.md

```diff
@@ -8,8 +8,8 @@ we are going to draft our pattern according to the measurements provided to us.
 
 Which begs the question, which measurements?
 
-It is you, as the pattern designer, who decides which measurements are used
-to draft your pattern. For our bib, the only measurement we need is the
+It is we, as the pattern designers, who decides which measurements are used
```

Comment:
We need to drop the `s` from `decides` here

Reply:
Addressed in PR #3410 .